### DISCUSSION TITLE
Failed integrity check  of moment.js(2.3.1) and moment-time.js()

### DISCUSSION THREAD

**Discussion by Ani4** [cdnjs/cdnjs#14226] [General]
I am having a issue while integrating 

`moment.js`

```<script src="https://cdnjs.cloudflare.com/ajax/libs/moment.js/2.30.1/moment.min.js" integrity="sha512-hUhvpC5f8cgc04OZb55j0KNGh4eh7dLxd/dPSJ5VyzqDWxsayYbojWyl5Tkcgrmb/RVKCRJI1jNlRbVP4WWC4w==" crossorigin="anonymous" referrerpolicy="no-referrer"></script>```

and 

`moment-timezone.js`
```<script src="https://cdnjs.cloudflare.com/ajax/libs/moment-timezone/0.5.44/moment-timezone.min.js" integrity="sha512-iIPUuTT6QbHMgNkbq2y0IudwnIfo34ln/5wmHwQ4obdZ2FM9HOLa+ERrcqSyODDfAmz4asR1hUnaaOYCOuBf1Q==" crossorigin="anonymous" referrerpolicy="no-referrer"></script>```

**Comment by MattIPv4** [comment]
Please see #14210
